secrethub_wrapper.go

```diff
@@ -24,37 +24,61 @@ struct SecretVersion {
 	time CreatedAt;
 	char* Status;
 };
+
+struct Client {
+	int ID;
```

Comment:
Is there a specific reason for not using `uint64`?

Reply:
Yes, because it is not supported on all architectures.

Reply:
Is it not though? Afaik the compiler works around this for 32 bit architectures.